index.html

```diff
@@ -2836,6 +2839,84 @@ <h2 class="icon-title">
           </div>
         </section>
       </section>
+      <section>
+        <h2>
+          Solid color icons
+        </h2>
+        <p>
+          Some platforms enforce that icons be displayed with a <dfn>solid
+          color</dfn>, where only the transparency of the icon can be
+          controlled. As web applications need to across multiple
+          platforms, it is possible to indicate that an icon can have a
+          user-agent-specified color applied by adding the <a>single-color</a>
+          purpose. This allows the platform to ensure that the icon looks well
+          integrated with the platform, and even apply different colors and
+          padding in different places throughout the platform.
+        </p>
+        <p>
+          When processing a <a>single-color</a> icon, the user agent MUST NOT use
+          the red component, green component, or blue component of a pixel. If
+          it has alpha equal to zero, the user agent SHOULD NOT display it.
+          If it the alpha component is greater than zero, the user agent SHOULD
+          display it with any tint.
+        </p>
+        <p>
+          If no single color icon is defined when an icon with a solid color is
+          expected to be presented, the user agent MAY show a fallback icon.
```

Comment:
It's unclear what this means. Does it mean it MAY fall back to using the standard icon? Or it MAY supply its own default icon?

I'm not sure we need to say this at all. User agents can show any icon they want wherever they want, and don't need permission of the spec. We should just focus on how user agents are expected to treat these monochrome icons, if supplied, and not start dictating their UI more broadly.

Reply:
MAY supply its own default icon. For example, in the PWA site controls notification image I give in the issue, the browser logo (Chrome) or PWA logo (Firefox) could continue to be used if the manifest has not supplied an icon with purpose: single-color

Reply:
Since maskable doesn't have any similar fallback text I'll remove this section.